### ISSUE TITLE
Accessibility: Screen reader not reading out dates on date picker

### ISSUE THREAD

**Issue by sgraham3311** [Hacker0x01/react-datepicker#1533]
### Expected behavior
When using screen reader the dates should be read out when navigating the dates in date picker using the arrow keys.
### Actual behavior
The date picker isn't announcing anything when used with JAWS 2018. Also, when using JAWS 2018, it doesn't reliably work with the arrow keys. You can go left and right, but moving up and down will sometimes move you out of the picker entirely.

Ideally it should be announcing all of the dates as you arrow key in any direction.

**Comment by Cicko** [comment]
We need this
